src/PowerShellEditorServices/Language/LanguageService.cs

```diff
@@ -294,72 +295,84 @@ public FindOccurrencesResult FindSymbolsInFile(ScriptFile scriptFile)
             ScriptFile[] referencedFiles,
             Workspace workspace)
         {
-            if (foundSymbol != null)
+            if (foundSymbol == null)
             {
-                int symbolOffset = referencedFiles[0].GetOffsetAtPosition(
-                    foundSymbol.ScriptRegion.StartLineNumber,
-                    foundSymbol.ScriptRegion.StartColumnNumber);
+                return null;
+            }
 
-                // Make sure aliases have been loaded
-                await GetAliases();
+            int symbolOffset = referencedFiles[0].GetOffsetAtPosition(
+                foundSymbol.ScriptRegion.StartLineNumber,
+                foundSymbol.ScriptRegion.StartColumnNumber);
 
-                // We want to look for references first in referenced files, hence we use ordered dictionary
-                var fileMap = new OrderedDictionary(StringComparer.OrdinalIgnoreCase);
-                foreach (ScriptFile file in referencedFiles)
-                {
-                    fileMap.Add(file.FilePath, file);
-                }
+            // Make sure aliases have been loaded
+            await GetAliases();
 
-                var allFiles = workspace.EnumeratePSFiles();
-                foreach (var file in allFiles)
+            // We want to look for references first in referenced files, hence we use ordered dictionary
+            var fileMap = new OrderedDictionary(StringComparer.OrdinalIgnoreCase);
```

Comment:
I know this is existing code, but wouldn't this be an issue on Linux?

Reply:
Yeah, this seems suspect...